core/tasks.py

```diff
@@ -154,13 +156,18 @@ def _run_scoring(score_job):
                 logger.info('Failed to successfully run image %s (%r)' % (score_algorithm_path, exc))
                 score_job.fail_reason = 'Return code: %s\nException:\n%r' % (result, exc)
             logger.info('Finished running image with result %r' % result)
-            # Store result
             score_result = ScoreResult(
                 score_job=score_job)
             score_result.data.save(
                 'score_job_%s.dat' % score_job.id, open(output_path, 'rb'))
             score_result.log.save(
                 'score_job_%s_log.dat' % score_job.id, open(stderr_path, 'rb'))
+            # one line code to extract the float from data file
+            score_result.overall_score = (float([line.strip() for line in score_result.data][0]))
+            # depends on the data type the return type varies
+            result_type = Enum('result_type', 'simple roc')
+            # for now, always return 'simple' as result type
+            score_result.result_type = (list(result_type)[0].name)
```

Comment:
I think we want a function to process this so to conceptually separate it.  In the future, it will be more complex.  For instance:
```
def _overall_score_and_result_type(datafile):
    # In the future, inspect the data to determine the result type.  For now, just extract a float from the data file
    result_type = ScoreResult.ResultType.SIMPLE
    overall_score = float(score_result.data.readline())
    return overall_score, result_type
```
and in this function
```
        score_result.overall_score, score_result.result_type = _overall_score_and_result_type(score_result.data)
```